pymc/distributions/censored.py

```diff
@@ -112,21 +108,8 @@ def rv_op(cls, dist, lower=None, upper=None, size=None, rngs=None):
         rv_out.tag.lower = lower
         rv_out.tag.upper = upper
 
-        if rngs is not None:
-            rv_out = cls._change_rngs(rv_out, rngs)
-
         return rv_out
 
-    @classmethod
-    def _change_rngs(cls, rv, new_rngs):
```

Comment:
What did this have to do with seeding?

Reply:
SymbolicDistributions required the ability to calculate the number of RNGs and reseed them in their graphs, because PyMC models were creating these variable states eagerly (and not letting them update after creation easily). Now that reseeding is done at compilation time, there is no need for these methods.